### PR TITLE
Kursad

### PR DESCRIPTION
**Pull request by kursadysr**
Please review my files.

### CODE REVIEW

**Comment by ozendizman** [comment]
Could be better, please push yourself harder..

**Review by ozendizman — CHANGES_REQUESTED** [review]
Next time try to do better.

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,4 @@ dist
 
 # TernJS port file
 .tern-port
+*.
```
Comment (ozendizman) [inline .gitignore:105]: Remove this sign "+"

File: .gitignore [hunk 2]
```diff
@@ -102,3 +102,4 @@ dist
 
 # TernJS port file
```
Comment (ozendizman) [inline .gitignore:103]: Could be better

**Review by ozendizman — APPROVED** [review]

**Review by mstylz1985 — APPROVED** [review]

**Pull request merged**